scripts/build.sh

```diff
@@ -1,50 +1,28 @@
 #!/usr/bin/env bash
```

Comment:
This change will affect TeamCity as the `build-test` script uses this

Reply:
yes, essentially adding an extra layer in the build script sandwich for TC

Reply:
TC currently calls `build.sh` which has changed drastically and does different things.

I'd prefer to not need to modify TC builds while adding Buildkite functionality.

Reply:
updated so TC --> build-test --> build-worker, buildkite --> pipeline --> build-worker